### PR TITLE
Fixes lab result order on dependent page

### PR DESCRIPTION
**Pull request by tiagograf-AOT**
# Fixes or Implements [AB#9652](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/9652)

* [ ] Enhancement
* [x] Bug
* [ ] Documentation

## Description

Fixes the lab result order by date on dependent page.

## Testing

* [ ] Unit Tests Updated
* [x] Functional Tests Updated
* [ ] Not Required

### Steps to Reproduce

If this is a bug, please provide details on how to reproduce the code.

### UI Changes

YES | NO

### Browsers Tested

* [ ] Chrome
* [ ] Safari
* [ ] Edge
* [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant.  Include any specialized deployment steps here.  

</br>
</br>
</br>
___
  
## Reviewer Guidelines

A reviewer should examine the PR if the Status above indicates the PR is **Ready**.
Once ready, the reviewer should read the information below as well as supporting information like the Azure ticket to ensure they have a good understanding of the request.

The reviewer should then:

1. Is the PR template complete and accurate?

    Confirm by reading all sections below.

1. Confirm the merge branch

    A normal feature/bug branch should merge into the Dev branch.  Exceptions for hotfix and sprint merges do occur and the target would then be Master.

1. Do the code commits have an appropriate comment and include linkage to the azure ticket by including a reference to "AB#nnnn"?

1. Review the Code Changes

    1. Do the changes match the description?

        The code commits should reflect the documented description and nothing more.  If additional changes are included the reviewer should call these out and request that the developer document the changes in Azure and in the description or remove the changed code.

    1. useful tips

        * Any obvious errors or omissions?
        * Does the code match the style of other similar work?
        * Is the code easy to understand?
        * Is the code well documented?
        * DB Migrations are appropriate?

If the Answer to any of the questions above is No, then a request for change should be the result of the review.

### CODE REVIEW

File: Apps/WebClient/src/ClientApp/src/components/dependentCard.vue [hunk 1]
```diff
@@ -86,6 +86,7 @@ export default class DependentCardComponent extends Vue {
             .then((results) => {
                 if (results.resultStatus == ResultType.Success) {
                     this.labResults = results.resourcePayload;
+                    this.sortEntries();
```
Comment (leobcgov) [inline Apps/WebClient/src/ClientApp/src/components/dependentCard.vue:89]: @ninosamson @tiagograf-AOT  It is good to sort the entries, but I don't see it as a part of the bug 9652.

File: Testing/functional/e2e/cypress/integration/dependent/dependents.js [hunk 2]
```diff
@@ -129,7 +129,7 @@ describe('dependents', () => {
     it('Validate Covid Tab with Results', () => {
         // Validate the tab and elements are present        
         cy.get('[data-testid=covid19TabTitle]').last().parent().click();
-        cy.get('[data-testid=dependentCovidTestDate]').first().should('have.text', ' 2020-08-21 ');
+        cy.get('[data-testid=dependentCovidTestDate]').first().should('have.text', ' 2020-10-03 ');
```
Comment (leobcgov) [inline Testing/functional/e2e/cypress/integration/dependent/dependents.js:132]: Should we add some steps to validate the sort is correct for these items?

**Review by leobcgov — CHANGES_REQUESTED** [review]

File: Apps/WebClient/src/ClientApp/src/components/dependentCard.vue (hunk 1 above)
Comment (tiagograf-AOT) [reply Apps/WebClient/src/ClientApp/src/components/dependentCard.vue:89]: The test was broken due to a bug that i fixed in the same PR, whats the problem here?

File: Testing/functional/e2e/cypress/integration/dependent/dependents.js (hunk 2 above)
Comment (tiagograf-AOT) [reply Testing/functional/e2e/cypress/integration/dependent/dependents.js:132]: I dont understand, it already does that, it checks for the first and the last element dates, whats your suggestion?

Comment (leobcgov) [reply Testing/functional/e2e/cypress/integration/dependent/dependents.js:132]: ah right, I didn't check the last item below

File: Apps/WebClient/src/ClientApp/src/components/dependentCard.vue (hunk 1 above)
Comment (leobcgov) [reply Apps/WebClient/src/ClientApp/src/components/dependentCard.vue:89]: sounds good, thanks @tiagograf-AOT

**Review by leobcgov — APPROVED** [review]

**Review by sslaws — APPROVED** [review]

**Pull request merged**